package.json

```diff
@@ -37,14 +37,14 @@
     "fastboot-app-server": "^3.3.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.2",
-    "@babel/eslint-parser": "^7.22.15",
-    "@babel/plugin-proposal-decorators": "^7.23.2",
+    "@babel/core": "^7.24.7",

```

Comment:
please remove this "^"

Reply:
Just for @babel/core right @iamitprakash

Reply:
@iamitprakash I have removed the ^ from the package.json, can you review it again.